File: WebShop_NULL/Models/ViewModels/AdminPanelModels/AdminPanelProductsViewModel.cs
using System.Collections.Generic;

namespace WebShop_NULL.Models.ViewModels.AdminPanelModels
{
    public class AdminPanelProductsViewModel//Moved to F# TODO test
    {
        public string Category = null;
        public string Query = null;
        public int FilterOption = 0;

        public IEnumerable<CategoryDTO> Categories;
        public IEnumerable<AdminPanelProductDTO> Products;
    }
}

File: sem2_FSharp/ViewModels/AdminPanelModels/AdminPanelProductsViewModel.fs
namespace sem2_FSharp.ViewModels.AdminPanelModels

open System.Collections.Generic
open sem2_FSharp.ViewModels

type AdminPanelProductsViewModel() =
    [<DefaultValue>]
    val mutable public Category: string
    
    [<DefaultValue>]
    val mutable public Query: string

    [<DefaultValue>]
    val mutable public FilterOption: int

    [<DefaultValue>]
    val mutable public Categories: IEnumerable<CategoryDTO>
    
    [<DefaultValue>]
    val mutable public Products: IEnumerable<AdminPanelProductDTO>
